### DISCUSSION TITLE
iii - proposal: exclude reported flips for long session score

### DISCUSSION THREAD

**Discussion by teavar** [idena-network/idena-docs#161] [IIPs]
iii - proposal: exclude reported flips from the long session score calculation, i.e. even if you choose the wrong side of the flip it won't affect your long session %

**Comment by ltraveler** [comment]
I wouldn't agree with that because whether a flip was reported or not, we have three levels of consensus:

- Strong consensus
- Weak consensus
- No consensus

Imo, there is no connection between the common sense of the network related to the flip and its reports. While a flip could be reported, we shouldn't conflate it with the common sense of the flip. If there is common sense, technically, that flip should have a score. If we are going to exclude them from the score, first of all, each mistake from non-reported flips would cost more, leading to a loss in the general score. Secondly, that mistake could even lead to the termination of an identity (The rule of session’s score should be 75% or more) in some cases where, without excluding the flips, the identity could maintain its status.

**Reply by teavar** [reply]
Which side of an absurd (no story) flip do you usually choose?

**Reply by teavar** [reply]
Or a flip where both sides can mean a story?

**Reply by ltraveler** [reply]
I think the type of flips where both sides could represent the story or where both sides do not have any meaning is only a particular case. Based on what I have seen during validation ceremonies, those types of flips were in the minority.

**Reply by teavar** [reply]
I've had plenty of them and usually they are the ones which lower my score, imho such flips should be excluded from the total score, but we'll need a menu or something similar where users will choose the reason for reporting the flip
